### DISCUSSION TITLE
fastMRI Leaderboard submission issue

### DISCUSSION THREAD

**Discussion by z-fabian** [facebookresearch/fastMRI#220] [Q&A] [answered]
Hi,

I have been trying to submit to the public leaderboard, but I always get the following error message:
```
UnboundLocalError: local variable 'request' referenced before assignment
```
I even tried to resubmit reconstructions that have already been accepted for the leaderboard in the past and received the same error.
Are there any recent changes in submission protocols?
Thanks!

**Comment by mmuckley** [comment, marked as answer]
I think Google might have updated their Drive API so that our old code doesn't work for this anymore.

Do you have another hosting service like Dropbox or an S3 bucket you could try? If we can isolate things to Google then I can create an issue to work on this later.

**Comment by z-fabian** [comment]
Thanks for the reply, I successfully submitted it via Dropbox. Fixing Google Drive submissions would be very helpful though!
